src/Streamly/Internal/Data/Stream/Channel/Types.hs

```diff
@@ -542,6 +546,14 @@ interleaved flag st = st { _interleaved = flag }
 getInterleaved :: Config -> Bool
 getInterleaved = _interleaved
 
+-- | Spawn bound threads (i.e., spawn threads using 'forkOS' instead of
+-- 'forkIO'). The default value is 'False'.
```

Comment:
Can you make a small TODO note here that this only takes effect for concurrent folds as of now?
And open an issue to implement support for `bound` everywhere.

Reply:
Sure; resolved in separate PR: https://github.com/composewell/streamly/pull/2406